docker-prune.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+# pruning is only run when inside CI to avoid accidentally removing stuff
+# GITHUB_ACTIONS is always set to true inside Github Actions
+# https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables
+if [ "${GITHUB_ACTIONS}" == true ] ; then
+  docker container prune -f
+  docker image prune -f
+  docker network prune -f
+  docker volume prune -f
+fi
```

Comment:
Same here, maybe move it to `.github`?

Reply:
Yes, it would be better to not show that to innocent eyes ;-)